src/base/bittorrent/session.h

```diff
@@ -570,7 +570,9 @@ namespace BitTorrent
         void applyBandwidthLimits(libtorrent::session_settings &sessionSettings);
 #else
         void configure(libtorrent::settings_pack &settingsPack);
-        void configurePeerClasses();
+        void initializeTypeFilter();
+        void configureLimitsOnLAN();
+        void configureLimitsOnUTP();
```

Comment:
I would join these two methods.

Reply:
+1
Do you have a name suggestion for the combination?

Reply:
configureBandwidthLimits?